Gemfile

```diff
@@ -14,7 +14,7 @@ gem 'puma', '~> 3.11'
 # Use Redis adapter to run Action Cable in production
 # gem 'redis', '~> 4.0'
 # Use ActiveModel has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
+#gem 'bcrypt', '~> 3.1.7'
```

Comment:
Any chance we can just back out all the 3-5 places where we've pulled the space out of a comment? It's not a huge deal, it's just kind of a readability anti-pattern.

Reply:
Can you explain more? I'm not sure I understand. What do you mean "back out"? Should there be a space between the hash and the word? If so, I can easily add that back in. . .

Reply:
Back out just meaning to undo the changes made. In this case, yeah, the space aids in readability.

Reply:
done